server.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+ //const express = require('express');
+const net = require('net');
+const uuid = require('uuid/v4');
+var server = require('http').Server();
+
+//const io = require('socket.io')(3000);
```

Comment:
In this lab we are converting what we did in previous labs to using socket.io, so line 7 here is correct and should be used.